packages/material-ui/src/styles/index.js

```diff
@@ -6,7 +6,7 @@ export { default as createStyles } from './createStyles';
 export { default as makeStyles } from './makeStyles';
 export { default as responsiveFontSizes } from './responsiveFontSizes';
 export { default as styled } from './styled';
-export * from './transitions';
+export { duration, easing } from './transitions';
```

Comment:
This exports the same methods as before but needed to change since `create` and `getAutoHightDuration` are now also named exports in `transitions`